flyway/migrations/V5__Create_SittingJobTable.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE SittingJob (
+    PK_jobID uniqueidentifier DEFAULT NEWSEQUENTIALID() PRIMARY KEY,
+    FK_sitterID uniqueidentifier NOT NULL,
+    FK_clientID uniqueidentifier NOT NULL,
+    startDate DATE,
+    startTime TIME,
+    endDate DATE,
+    endTime TIME,
```

Comment:
Does not match ERD.

Should be:
![image](https://github.com/House-and-Pet-Sitting-Database-Levelup/HouseAndPetSittingDB/assets/125252284/ea89a96e-7750-422e-aca4-555d53b74314)
StartDateTime(DATETIME type)
EndDateTime(DATETIME type)